src/amo/components/SearchPage.js

```diff
@@ -0,0 +1,36 @@
+import React, { PropTypes } from 'react';
+
+import Paginate from 'core/components/Paginate';
+import { gettext as _ } from 'core/utils';
+
+import SearchResult from './SearchResult';
+import SearchForm from 'core/components/Search/SearchForm';
+import SearchResults from 'core/components/Search/SearchResults';
+
+
+export default class SearchPage extends React.Component {
+  static propTypes = {
+    count: PropTypes.number,
+    loading: PropTypes.bool.isRequired,
+    page: PropTypes.number,
+    results: PropTypes.array,
+    query: PropTypes.string,
+  }
+
+  render() {
+    const { count, loading, page, query, results } = this.props;
+    // TODO: Don't hardcode this.
+    const pathname = '/en-GB/firefox/search/';
+    const paginator = query && count > 0 ?
+      <Paginate count={count} pathname={pathname} query={{ q: query }} currentPage={page} /> : [];
```

Comment:
That should probably be `null` instead of `[]`. I'm guessing that isn't changed though...